openapi.go

```diff
@@ -95,6 +95,11 @@ func declareAllTagsFromOperations(s *Server) {
 // Also serves a Swagger UI.
 // To modify its behavior, use the [WithOpenAPIConfig] option.
 func (s *Server) OutputOpenAPISpec() openapi3.T {
+	s.OpenAPI.Description().Servers = append(s.OpenAPI.Description().Servers, &openapi3.Server{
+		URL:         s.proto() + "://" + s.Addr,
```

Comment:
there should be a method on `Server` for computing `s.proto() + "://" + s.Addr`

Reply:
It's also used in

https://github.com/go-fuego/fuego/blob/main/serve.go

Reply:
OIC in the startup messages. Will update in a second

Reply:
Adding this would have been appreciated
```
    Co-authored-by: ccoVeille <3875889+ccoVeille@users.noreply.github.com>
```

Reply:
I don't have push rights to main. If @EwenQuim wouldn't mind doing it that would be great.